sklearn/cluster/hierarchical.py

```diff
@@ -829,6 +833,18 @@ class FeatureAgglomeration(AgglomerativeClustering, AgglomerationTransform):
         are merged to form node `n_features + i`
     """
 
+    def __init__(self, n_clusters=2, affinity="euclidean",
+                 memory=None,
+                 connectivity=None, compute_full_tree='auto',
+                 linkage='ward', pooling_func=np.mean):
+        self.n_clusters = n_clusters
```

Comment:
And could you please use `super(FeatureAgglomeration, self).__init__(n_clusters=..., ...)` instead of setting things individually here?

Reply:
Resolved.